placeholderkv.conf

```diff
@@ -199,15 +199,15 @@ tcp-keepalive 300
 # server to connected clients, masters or cluster peers.  These files should be
 # PEM formatted.
 #
-# tls-cert-file redis.crt
-# tls-key-file redis.key
+# tls-cert-file placeholderkv.crt
+# tls-key-file placeholderkv.key
```

Comment:
Where possible, let's make these agnostic to the name. 
```suggestion
# tls-cert-file server.crt
# tls-key-file server.key
```

Reply:
Agreed.